### DISCUSSION TITLE
Can a single window app skip checking the window ID?

### DISCUSSION THREAD

**Discussion by DerekSauer** [rust-windowing/winit#3152] [Q&A] [answered]
Hello,

I'm working on an application that is only going to have one rendering window. Is it safe to assume that any WindowEvent sent to the app from the OS is destined for that window and checking WindowID inside the WindowEvent handler is unnecessary?

**Comment by kchibisov** [comment, marked as answer]
It should be safe.

**Reply by DerekSauer** [reply]
Great, thank you very much.
